### PR TITLE
upgraded easy vulnerable deps

### PR DESCRIPTION
**Pull request by orndorffgrant**
If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
Upgrades the easy to upgrade dependencies that github says are vulnerable.
#### Related JIRA tickets:
ORANGE-1047
#### How should this be manually tested?
make sure the tests run and the server starts up. Try logging in.
#### Screenshots (if appropriate):

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,8 +82,7 @@
     "babel-preset-es2015": "6.16.0",
     "babel-preset-stage-2": "6.18.0",
     "chai-datetime": "1.5.0",
-    "codacy-coverage": "2.0.2",
```
Comment (rmharrison) [inline package.json]: Good catch on removing `codacy-coverage`

File: package.json [hunk 2]
```diff
@@ -56,14 +55,14 @@
...
-    "method-override": "^2.3.5",
+    "method-override": "3.0.0",
     "moment": "2.19.3",
     "morgan": "1.9.0",
     "nodemailer": "4.1.0",
     "owasp-password-strength-test": "1.3.0",
     "passport": "0.4.0",
     "passport-facebook": "2.1.1",
-    "passport-jwt": "3.0.0",
+    "passport-jwt": "false4.0.0",
```
Comment (rmharrison) [inline package.json]: Minot: Should this be `false` or `^`?

**Comment by rmharrison** [comment]
@orndorffgrant : I dislike the github GUI for resolving merge conflicts.

Recommend merging from command-line, and including the updated `yarn.lock` in the merge commit.

Can use `git merge --ours` or manually resolve.

git's default merge commit doesn't include the PR when merging from command line, so you'll have update the commit message manually. For this PR

> Merge pull request #81 from amida-tech/ask/orange-1047-update-vuln-deps-take-two

Both the PR and branch differ by repo, so you **cannot copy/paste** the merge commit message exactly.

File: package.json (hunk 2 above)
Comment (orndorffgrant) [reply package.json]: yup thanks

File: package.json [hunk 3]
```diff
@@ -45,7 +45,6 @@
     "eslintrc": "^1.0.6",
     "express": "4.16.0",
     "express-actuator": "^1.1.0",
-    "express-jwt": "5.3.0",
     "express-jwt-permissions": "1.2.1",
```
Comment (rmharrison) [inline package.json]: Since `express-jwt` is being removed, is `express-jwt-permissions`  removed as well?

Comment (orndorffgrant) [reply package.json]: nope, despite the name similarity, the latter is not dependent on the former, and we are using it.

**Comment by orndorffgrant** [comment]
@rmharrison 

I usually prefer merging develop into the PR branch, resolving conflicts, and making sure everything's okay (from command-line).

Then we can use Github's fun merge button which will give us the nice message automatically.

**Pull request merged**